src/components/profile/statistics/Statistics.tsx

```diff
@@ -34,57 +25,32 @@ export const Statistics = ({
           : t("statistics.titleSection")}
       </h2>
       <BadgeSubojectivesCategoryCompletion
-        themeToCompare="practices"
+        label={skillsDatas?.find((skill) => skill.label === "practices").label}
```

Comment:
```suggestion
        label={"practices"}
```
We're searching for a skill which has the label "practices" and take the value of its label.
Obviously it will be "practices". So can write it directly

Reply:
omg I'm completely matrixed by .find 💩thanks to have noticed that !